lib/route_types.txt

```diff
@@ -81,6 +81,7 @@ ZEBRA_ROUTE_BGP_DIRECT_EXT, bgp-direct-to-nve-groups, NULL, 'e', 0, 0, 0, "BGP2V
 ZEBRA_ROUTE_BABEL,      babel,     babeld, 'A', 1, 1, 1,     "Babel"
 ZEBRA_ROUTE_SHARP,      sharp,     sharpd, 'D', 1, 1, 1,     "SHARP"
 ZEBRA_ROUTE_PBR,        pbr,       pbrd,   'F', 1, 1, 0,     "PBR"
+ZEBRA_ROUTE_BFD,        bfd,       bfdd,   'f', 1, 1, 0,     "BFD"
```

Comment:
Should we consider adding a new column type to be read in about something that should not show up in the `show ip route` header?

Reply:
Fixed: I just disallowed it in IPv4 and IPv6 and now it doesn't show up in `show ip route` anymore.